main.py

```diff
@@ -1 +1,10 @@
-print("Hello World")
\ No newline at end of file
+def isPalindrome(word):
+
+    modified_string="".join(char.lower() for char in word if char.isalnum())
+    if modified_string == modified_string[::-1]:
+        return True
+    return False
```

Comment:
There is probably a better way to do this. Why not try the following , instead of doing an if-else statement try:

```suggestion
return string == string[::-1]
```